src/extension.ts

```diff
@@ -17,6 +17,12 @@ export function activate(context: vscode.ExtensionContext) {
         }
     });
 
+    vscode.workspace.onDidChangeConfiguration((event: vscode.ConfigurationChangeEvent) => {
+        if (event.affectsConfiguration(maxEntriesPerFile)) {
+            provider.maxEntriesPerFile();
```

Comment:
The method name `maxEntriesPerFile` is not descriptive enough.
Instead, we should name it `updateMaxEntriesPerFile()`.